pyproject.toml

```diff
@@ -24,6 +24,8 @@ tifffile = "^2022.10.10"
 matplotlib = "3.4.3"
 pillow = "~9.1"
 scikit-image = "^0.19.3"
+xarray = "^2022.11.0"
```

Comment:
Is this what takes the place of the requirements file? I know we have pretty strict requirements in `ark`, I'm wondering if it's possible to make the `tmi` requirements more flexible in case other people want to import any of these, while keeping the ones in our own respos more restrictive.

Reply:
kk, can do that in a later PR